src/services/DhService.ts

```diff
@@ -303,69 +283,23 @@ export abstract class DhService<TDH = unknown, TClient = unknown>
       return;
     }
 
-    const changed = [...result!.changes.created, ...result!.changes.updated];
-
-    // Have to set this with type assertion since TypeScript can't figure out
-    // assignments inside of the `forEach` and will treat `lastPanel` as `null`.
-    let lastPanel = null as vscode.WebviewPanel | null;
-
-    changed.forEach(({ id, title = 'Unknown', type }) => {
-      assertIsVariableID(id, 'id');
+    const changed = [
+      ...result!.changes.created,
+      ...result!.changes.updated,
+    ] as VariableDefintion[];
```

Comment:
Would `satisfies VariableDefintion[]` work here? Not sure why the casting is necessary.

Reply:
Unfortunately no. It complains about the mismatch of the branded ids with string ids. I added a comment.